EIPS/rentcollector.md

```diff
@@ -0,0 +1,67 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Rent Collection for Blockchain Data
+    Author: Daniel A. Nagy <daniel@ethereum.org>, Zsolt Felfoldi <zsolt@ethereum.org>
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-11-01
+
+
+## Simple Summary
+Introducing a way to collect rent from accounts that is proportional 
+to the cost that they impose on the network by storing a large state
+over a long period of time.
+
+## Abstract
+A new global parameter called `rent` similar to `gasLimit` that can be 
+modified by miner vote is introduced. It is initialized at 0 and is 
+assumed to have been zero before its introduction. The miner validating 
+a particular block can increase rent in a bounded way at the cost of 
+forfeiting a proportional part of the mining reward. Similarly, if 
+`rent` is more than zero, one can decrease it in a bounded way and 
+receive the same additional mining reward that would need to be 
+forfeited to raise it back.
+
+With each new block, an account is chosen in a deterministically random 
+fashion with a probability proportional to the amount of storage that 
```

Comment:
How do you enforce deterministically random choosing of an account with storage on it? This seems like a detail that's being handwaved over, when there may be a means to create an economic incentive to focus on "useless" accounts (accounts that have are outdated, low use, etc.)